src/core/server/core_route_handler_context.ts

```diff
@@ -103,10 +104,30 @@ class CoreUiSettingsRouteHandlerContext {
   }
 }
 
+class CoreDeprecationsRouteHandlerContext {
```

Comment:
Can we update `src/core/server/core_route_handler_context.test.ts` to include tests for this?